sway-core/src/semantic_analysis/namespace/module.rs

```diff
@@ -67,6 +72,7 @@ impl Default for Module {
             name: Default::default(),
             span: Default::default(),
             is_external: Default::default(),
```

Comment:
This won't work without having a circular reference to the root Module inside each Module to get access to the `mod_path`